src/main/java/core/basesyntax/zoo/Penguin.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.zoo;
+
+public class Penguin extends Bird implements Swimmable {
```

Comment:
birds are fliable. so penguin is fliable. that's wrong :)

Reply:
I don`t agree. Penguin is a bird but it can`t fly only swim